tests/e2e/verification/test_start_verification.py

```diff
@@ -0,0 +1,49 @@
+from sinch.domains.verification.models.responses import StartVerificationResponse
+
+
+def test_start_verification_sms(
+    sinch_client_sync,
+    phone_number
+):
+    verification_response = sinch_client_sync.verification.verifications.start(
+        method="sms",
+        identity={
+            "type": "number",
+            "endpoint": phone_number
+        },
+        reference="random"
+    )
+
+    assert isinstance(verification_response, StartVerificationResponse)
```

Comment:
Wouldn't it be better to have a dedicated model for `sms` (same for the 3 other type) as described in the OAS document? So that when getting a response from this request, the end user has access only to the properties returned by the server.
For reference, it has been done here in java: https://github.com/sinch/sinch-sdk-java/blob/main/client/src/main/com/sinch/sdk/domains/verification/adapters/converters/VerificationsDtoConverter.java#L90

Reply:
I was planning to ask for same.
Better added value for end user to have dedicated models defined

Reply:
Done 👌

Reply:
Nice, now is it also possible to update the test in order to verify the response is an instance of a subclass (is should be `StartSMSInitiateVerificationResponse` if I understand well the inheritance in Python?) instead of the parent `StartVerificationResponse` ?

Reply:
fixed